tests/tests/operation_tests.cpp

```diff
@@ -26,6 +28,22 @@ using namespace golos::protocol;
 using std::string;
 
 
+namespace boost { namespace container {
+
+template<typename T>
+std::ostream &operator<<(std::ostream &out, const flat_set<T> &t) {
```

Comment:
We have other files with tests (`proposal_tests.cpp`, etc…), where this function becomes required to print error messages. Please, move this (in one of next PRs) so it can be accessible from other files. Maybe to `database_fixture.hpp` (but it's already quite overloaded) or maybe to new header